site/content/en/docs/start/_index.md

```diff
@@ -425,20 +425,19 @@ choco install minikube
 {{% quiz_instruction id="/Windows/x86-64/Stable/.exe download" %}}
 1. Download the [latest release](https://storage.googleapis.com/minikube/releases/latest/minikube-installer.exe).  
 <br>
-    Or if you have `curl` installed, use this command:
-    ```shell
-    curl -Lo minikube.exe https://github.com/kubernetes/minikube/releases/latest/download/minikube-windows-amd64.exe
-    New-Item -Path "c:\" -Name "minikube" -ItemType "directory" -Force
-    Move-Item .\minikube.exe c:\minikube\minikube.exe -Force
+    Or if using `PowerShell`, use this command:
+    ```powershell
+    New-Item -Path 'c:\' -Name 'minikube' -ItemType Directory -Force
+    Invoke-WebRequest -OutFile 'c:\minikube\minikube.exe' -Uri 'https://github.com/kubernetes/minikube/releases/latest/download/minikube-windows-amd64.exe' -UseBasicParsing
     ```
 
 2. Add the binary in to your `PATH`.  
 <br>
     _Make sure to run PowerShell as Administrator._
```

Comment:
Could you move this tip to above step 1 so the user knows to use admin from the start.

Reply:
An Administrator command prompt isn't necessary to run the commands in Step 1. It can be completed as a normal user. Whereas Step 2, it's required.

An install script could be built to do all the environment checks and error checking. Similar to how [Chocolatey is installed](https://chocolatey.org/install).

```powershell
Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
```

If this would be of interest, I'll volunteer to build `v0.0.0.1-alpha-preview001`. 🙂

Reply:
I understand it's only required for step 2, but for user convenience it's better if we tell them to use Powershell as admin at the start of the instructions. Otherwise they may open Powershell in non-admin mode and then when they get to step 2 they see they need to open in admin mode and they have to open another instance.

Reply:
I don't think this is a good approach. The technical audience of this product should be comfortable launching different prompts. Especially considering a new prompt must be launched after modifying `$env:PATH`. The current documentation is accurate and requires minimal commands to get everything initialized.

If convenience is the goal, an install script should be used similar to the Chocolatey install. However, that could be considered an over-engineered solution to what is currently a simple process and snowball into undesired maintenance. Take for example my current system which is running WS2022 Preview. The certificate store has a limited set of [trusted certificates](https://techcommunity.microsoft.com/t5/windows-server-insiders/server-2022-preview-missing-let-s-encrypt-root-certificate/m-p/2617629) requiring extra steps since the certificate presented by GitHub isn't trusted. By taking the convenience route, anomalies such as this must be accounted for in the installation script. Making the install script prone to bloat as well as the FAQ documenting these one-off scenarios.

The current documentation is the definition of concise. Convenience is an installation script. My aim with this PR is to provide the native PowerShell commands. 🙂

Reply:
All I'm requesting is the first line is reworded from:
`Or if using PowerShell, use this command:`
to
`Or install via Powershell using the following commands (Make sure to run PowerShell as Administrator):`
Or something similar, I think it's best if we make the requirements clear at the start of the command instructions. Just like the `What you’ll need` right above the install instructions.